### PR TITLE
Update the examples readme to include vercel deployment information

### PR DESCRIPTION
**Pull request by frederikprijck**

### CODE REVIEW

**Review by adamjmcgrath — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 1]
```diff
@@ -25,4 +25,59 @@ AUTH0_ISSUER_BASE_URL=https://YOUR_AUTH0_DOMAIN
...
+| AUTH0_AUDIENCE | Plaintext | YOUR_AUTH0_API_IDENTIFIER |
+| AUTH0_SCOPE | Plaintext | openid profile read:shows
+
+#### Configuring Auth0
+
+Go to the Application Settings on your [Auth0 dashboard](https://manage.auth0.com/) and make sure to configure the following:
+
+| Setting               | Description                                                                                                                                                            |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Allowed Callback URLs | Should be set to `https://nextjs-auth0-*.vercel.app/api/auth/callback` when deploying to vercel. |
```
[the remark this answers is not in the document]
Comment (frederikprijck) [reply examples/README.md:80]: In my Vercel app, prod also has a suffix:

image

File: examples/README.md [hunk 2]
```diff
@@ -25,4 +25,59 @@ AUTH0_ISSUER_BASE_URL=https://YOUR_AUTH0_DOMAIN
...
+
+#### Configuring Vercel
+If you do not have a vercel account yet, move over to https://vercel.com/ to sign up for one.
+Once logged in to your account, you can create a new project and import a Git repository.
+
+Vercel should automatically select the `Next.js` Framework Preset.
+Because our deployment is a bit different from a standard Next.js repository, we will need to override the `Build and Output settings`:
+
+- Build Command: `npm run build:vercel`
+- Output Directory: `examples/kitchen-sync-example/.next`
```
[the remark this answers is not in the document]
Comment (frederikprijck) [reply examples/README.md:44]: Oh my god, I will never learn it 😂

File: examples/README.md [hunk 3]
```diff
@@ -25,4 +25,59 @@ AUTH0_ISSUER_BASE_URL=https://YOUR_AUTH0_DOMAIN
 AUTH0_BASE_URL=http://localhost:3000/
 AUTH0_CLIENT_ID=YOUR_AUTH0_CLIENT_ID
 AUTH0_CLIENT_SECRET=YOUR_AUTH0_CLIENT_SECRET
+AUTH0_SCOPE=openid profile read:shows
+AUTH0_AUDIENCE=YOUR_AUTH0_API_IDENTIFIER
 ```
+
+### Hosting on Vercel
```
[the remark this answers is not in the document]
Comment (frederikprijck) [reply examples/README.md:31]: Done

File: examples/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (frederikprijck) [reply examples/README.md]: Fixed it

File: examples/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (frederikprijck) [reply examples/README.md]: Updated to use {YOUR_VERCEL_PROJECT_NAME}

**Review by adamjmcgrath — APPROVED** [review]

**Pull request merged**